### ISSUE TITLE
Error 500 out of the box

### ISSUE THREAD

**Issue by vuokko** [kristapsdz/kcgi-framework#3]
I'm running OpenBSD 6.6 with patches and kcgi and ksql are installed from packages, web server running and everything should be fine.

After changing kwebap to ort in makefile, building, seeding user info and installing, Trying to log in, login.json generates error 500 for good or bad credentials.

What and how to debug more?

**Comment by kristapsdz** [comment]
I've made some extensive changes to bring the system up to date.  Does it work for you now?

**Comment by vuokko** [comment]
I got it running. The sqlbox was little surprise as it isn't available as package on OpenBSD 6.6 stable. Compiling and installing was easy for it.

- ort-c-source wanted -s to get rid of warnings in Makefile
- extern.h didn't knew struct ksqlstmt.
- db_open_logging wasn't anywhere to be found. Is it new feature?
- To Link the program -lksql was needed.

After those modifications it worked and I gave git diff>quick_changes Feel free to adapt wanted features.
[quick_changes.txt](https://github.com/kristapsdz/kcgi-framework/files/4264342/quick_changes.txt)
